utils/ansible/ftest/file/configure-nofile.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# set -x
+set -e -o pipefail
```

Comment:
`-u` is another good flag to add so that you don't have variables that are unexpectedly unassigned going unchecked for assignment.  It does mean your code needs to use `${varname:-}` where you intentionally want to allowa  variable to be legitimately unassigned.